.circleci/images/chrome/beta/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM circleci/node:latest
+FROM cimg/node:current-browsers
```

Comment:
Just to double check, what is the version of node in this image?

Reply:
17.6.0 is the current version.